src/index.ts

```diff
@@ -1,4 +1,4 @@
-export {RippleAPI} from './api'
+export * from './api'
```

Comment:
was this intentional? Seems like a bigger change than the PR title conveys

Reply:
I think there was some other stuff in ripple-lib that I want to access, but you're right, that should go in a separate PR.